activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/src/main/java/org/activiti/cloud/services/events/converter/VariableCreatedEventConverter.java

```diff
@@ -35,8 +35,8 @@ public ProcessEngineEvent from(ActivitiEvent event) {
                                             event.getProcessDefinitionId(),
                                             event.getProcessInstanceId(),
                                             ((ActivitiVariableEventImpl) event).getVariableName(),
-                                            ((ActivitiVariableEventImpl) event).getVariableValue().toString(),
-                                            ((ActivitiVariableEventImpl) event).getVariableType().getTypeName(),
+                                            String.valueOf(((ActivitiVariableEventImpl) event).getVariableValue()),
```

Comment:
I know that it was like this before, but why do we need to convert to string here?

Reply:
@erdemedeiros damn.. we need a test for this one as well. I've changed this because I was getting a NPE when the variable didn't existed. Before it was ((ActivitiVariableEventImpl) event).getVariableValue().toString(), which caused the NPE when ((ActivitiVariableEventImpl) event).getVariableValue() was null.